config/schemas/config.yaml

```diff
@@ -5344,6 +5353,26 @@ properties:
         $ref: '#/$defs/kubernetesNodeSelector'
       affinity:
         $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+      sealedSecrets:
+        properties:
+          enabled:
+            default: false
+            type: boolean
+      mongodb:
+        properties:
+          enabled:
+            default: false
+            type: boolean
+      fluxv2:
+        properties:
+          enabled:
+            default: false
+            type: boolean
+      kafka:
+        properties:
+          enabled:
+            default: false
+            type: boolean
```

Comment:
```suggestion
```
This should not be configured under `certmanager`.

Reply:
I removed them.